### PR TITLE
added stuff to readme

### PR DESCRIPTION
**Pull request by CodyNicholson**
Hey, do you like my work?

### CODE REVIEW

**Comment by CodyNicholson** [comment]
Hey, this is stupid

**Comment by zacharyd97** [comment]
it's 👍

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# Demo_Project
\ No newline at end of file
+# Demo_Project
+
+By: Cody
```
Comment (CodyNicholson) [inline README.md:3]: Should we check for invalid input?

**Comment by Officialyi** [comment]
No conflicts, approve for merge

File: README.md (hunk 1 above)
Comment (zacharyd97) [reply README.md:3]: Looks good

**Review by zacharyd97 — APPROVED** [review]

**Pull request merged**